examples/microwave-oven-app/microwave-oven-common/src/microwave-oven-device.cpp

```diff
@@ -29,20 +29,20 @@ Protocols::InteractionModel::Status
 MicrowaveOvenDevice::HandleMicrowaveOvenSetCookingParametersCallback(uint8_t cookMode, uint32_t cookTime, uint8_t powerSetting)
 {
     // placeholder implementation
-    // TODO: set Microwave Oven cooking mode by cookMode.Value().
+    // TODO(#30609): set Microwave Oven cooking mode by cookMode.Value().
     mMicrowaveOvenControlInstance.SetCookTime(cookTime);
     mMicrowaveOvenControlInstance.SetPowerSetting(powerSetting);
     return Status::Success;
 }
 
-Protocols::InteractionModel::Status MicrowaveOvenDevice::HandleMicrowaveOvenAddMoreTimeCallback(uint32_t addedCookTime)
+Protocols::InteractionModel::Status MicrowaveOvenDevice::HandleMicrowaveOvenAddMoreTimeCallback(uint32_t finalCookTime)
```

Comment:
This should be called "HandleMicrowaveSetCookTimeCallback" since it takes the final time always, and it doesn't actually "add more time", but rather "sets the time".

Reply:
But usually the callback function name should be associated to the cluster commands name? Would change to "HandleMicrowaveOvenAddMoreTime**Command**Callback" is better?

Reply:
> This should be called "HandleMicrowaveSetCookTimeCallback" since it takes the final time always, and it doesn't actually "add more time", but rather "sets the time".

Do the callback methods name should be associated to the cluster commands name usually?

Reply:
No. The API for the cluster has to be "usable" and understandable. It does not have to be 1:1 matching cluster name. In the microwave case, there is only 1 version of time: the time left. When you add time, you increase time left. When you set cook time before start, you directly set time left.